app/src/main/java/com/globant/example/mentorapp/domain/UserPresenterImpl.java

```diff
@@ -0,0 +1,57 @@
+package com.globant.example.mentorapp.domain;
```

Comment:
Presenters are not part of the domain layer.

MVP targets presentation layer, so a module looks like this:

```
* feature
  --> domain
      --> interactor
      --> model (data model)
  --> presentation
      --> model (view model)
      --> presenter
      --> view

```